src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -180,7 +180,8 @@ def _get_hosts_from_network(net):
         except IndexError:
             return []
 
-    def wait_till_nodes_are_ready(self, network_name):
+    def wait_till_nodes_are_ready(self, network_name: str = None):
+        assert network_name is not None
```

Comment:
sounds weird to have a default that will raise, no?

Reply:
I couldn't find any clean solution, I will leave this part to the next refactoring where the day2 cluster will spawned in its own terraform stack.